win/libsass.sln

```diff
@@ -1,10 +1,21 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libsass", "libsass.vcxproj", "{E4030474-AFC9-4CC6-BEB6-D846F631502B}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".SolutionItems", ".SolutionItems", "{33318C77-2391-4399-8118-C109155A4A75}"
+	ProjectSection(SolutionItems) = preProject
+		..\.editorconfig = ..\.editorconfig
```

Comment:
It seems to be working. I installed [editorconfig for VS](https://github.com/editorconfig/editorconfig-visualstudio) extension, opened `libsass/win/libsass.sln`, set `indent_size` to 12, opened `libsass/src/file.cpp` and Ctrl+K,D to format the document,  the intended 12 space indentation applied.

Note starting with VS2017, editorconfig and cmake have built-in support [[ref](https://www.visualstudio.com/en-us/news/releasenotes/vs2017-relnotes)].